backend/frontend/tests/test_restricted_user_board_access.py

```diff
@@ -114,20 +116,23 @@ def test_user_without_view_access_cannot_view_other_user_board(self):
         does not have access to the user board.
         """
         self.do_login(self.user_without_access)
-        url = self.reverse_url("user_board", kwargs={"user_id": self.viewed_user.id})
+        url = self.reverse_url("user_board", kwargs={
+                               "user_id": self.viewed_user.id})
 
         self.page.goto(url)
 
         body = self.page.text_content("body")
-
-        self.assertIn("You don't have permission to access this page", body)
+        
+        self.assertIn("You don't have permission to view this page.", body)
```

Comment:
For consistency's sake, let's check for the main h1 text "Permission Denied" instead. The one being used currently seems very random and could change, even though it's highly unlikely.